src/hooks/useLocalNotifications.tsx

```diff
@@ -109,6 +111,7 @@ const useLocalNotificationsContext = () => {
               status: currentStatus,
               isExchange,
               requestId,
+              tracked,
```

Comment:
Not sure I see when this get set, should it be set after a successful `trackSkipTx` call?